Serverless installation updates for 'serverless.ts' and trace merging (#13199)
DataDog/documentation: content/en/serverless/installation/python.md

    plugins:
      - serverless-plugin-datadog
    ```

> <div class="alert alert-info">If you are instead deploying your Serverless Framework app <a href="https://www.serverless.com/framework/docs/providers/aws/guide/intro">by natively exporting a JSON object from a Javascript file</a> (e.g. using a "serverless.ts" file), follow our <a href="https://docs.datadoghq.com/serverless/installation/python?tab=custom"> custom installation instructions</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-info">If you are instead deploying your Serverless Framework app <a href="https://www.serverless.com/framework/docs/providers/aws/guide/intro">by natively exporting a JSON object from a JavaScript file</a> (for example, by using a <code>serverless.ts</code> file), follow the <a href="https://docs.datadoghq.com/serverless/installation/python?tab=custom">custom installation instructions</a>.</div>
```
